README.md

```diff
@@ -48,6 +48,12 @@ import { Generator } from '@jspm/generator';
 
 const generator = new Generator({
   mapUrl: import.meta.url,
+  inputMap: {
+    "imports": {
+      "react-dom": "https://ga.jspm.io/npm:react-dom@17.0.2/index.js"
+    },
+    "scopes": {/* ... */}
+  },
```

Comment:
Can we rather move this one to the end of the example options as a more "advanced" feature. Should we demonstrate a non-jspm mapping?

Reply:
yeah let's remove it

Reply:
> Should we demonstrate a non-jspm mapping?

This is a really good idea! Let me do that

Reply:
Ah sorry I misread, yeah I was considering making that a separate example but the use-case seems so mainstream I decided to modify the main one. I can barely imagine people always creating their import maps from scratch. But happy to change if you think otherwise

Reply:
Yeah I guess so. It would also be nice to support eg `/react.js` where it is assumed the URL is relative to the rootURL for the input map when rootURL is set. This way we can support `/` normalization not just turning into the file system base. But that's for a follow-on.